templates/prometheus_exporter/solrcloud_prometheus_exporter.service.j2

```diff
@@ -0,0 +1,19 @@
+[Unit]
```

Comment:
This service is not part of the role any more, unneeded file

Reply:
right! Deleted